dddwa/dddperth-website
# Update 2021 logo in og image and about page

Updates the OG Image and about image to use the 2021 version of the
DDD logo

### Testing
* View localhost:3000/about
* The logo is updated to the 2021 version next to the regular ddd logo
* Open up dev tools and search for the og:image
* It should be updated to the 2021 logo
